content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -485,6 +485,25 @@ The following configuration variables are for features that are available for us
 : Enables improved resource names for web spans when set to `true`. Uses route template information where available, adds an additional span for ASP.NET Core integrations, and enables additional tags. Added in version 1.26.0. Enabled by default in 2.0.0<br>
 **Default**: `true`
 
+### Headers extraction and injection
+
+Datadog APM tracer supports [B3][9] and [W3C][10] headers extraction and injection for distributed tracing.
+
+Distributed headers injection and extraction is controlled by configuring injection/extraction styles. Currently two styles are supported:
```

Comment:
Datadog, B3, and W3C = 3?
```suggestion
Distributed headers injection and extraction is controlled by configuring injection/extraction styles. Currently three styles are supported:
```

Reply:
Wait 3? :mindblown: I'll learn how to count.
Fixed in 201b20b2e5bcf014ab433ee628af8fe84c7faf49 (i've actually set 4, cause I assume we should count B3 single as well